staging_service/autodetect/Mappings.py

```diff
@@ -29,6 +29,25 @@
 
 MEDIA = "KBaseBiochem.Media"
 
+fastq_reads_id = "fastq_reads"
```

Comment:
Are all these new fields tested in any way? Do they not need to be or something?

Reply:
They map to a json with a list of mappings in the narrative service. Not sure how we would test that